cloud/google/node_init.sh

```diff
@@ -6,4 +6,21 @@ sudo mkdir ${DATA_DIR}
 sudo /usr/share/google/safe_format_and_mount \
   -m "mkfs.ext4 -F" ${DATA_DEV} ${DATA_DIR}
 
-
+# Install google-fluentd which pushes application log files up into the Google
+# Cloud Logs Monitor.
+curl -sSO https://storage.googleapis.com/signals-agents/logging/google-fluentd-install.sh
```

Comment:
This fetch & install stuff just comes verbatim from the docs:
https://cloud.google.com/logging/docs/agent/installation